src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -1446,8 +1457,26 @@ function addRubricCol(questionNumber) {
                                       'rubricChoice');
 
     // Insert after last <th>
-    var lastTh = $('#rubricEditTable' + idSuffix + ' th:last');
+    var lastTh = $('#rubricEditTable' + idSuffix).find('tr:first').children().last();
     $(rubricHeaderFragment).insertAfter(lastTh);
+    
+    // Insert weight <th>
+    var rubricWeightFragmentTemplate = 
+        '<th class=\"rubricCol-${qnIndex}-${col}\">'
+       +    '<input type=\"number\" class=\"form-control nonDestructive\" value=\"${rubricWeight}\" id=\"${Const.ParamsNames.FEEDBACK_QUESTION_RUBRIC_WEIGHT}-${qnIndex}-${col}\" name=\"${Const.ParamsNames.FEEDBACK_QUESTION_RUBRIC_WEIGHT}-${col}\" step=\"0.01\">'
+       +'</th>';
+
+    var rubricWeightFragment = rubricWeightFragmentTemplate;
+    rubricWeightFragment = replaceAll(rubricWeightFragment, '${qnIndex}', questionNumber);
+    rubricWeightFragment = replaceAll(rubricWeightFragment, '${col}', newColNumber - 1);
+    rubricWeightFragment = replaceAll(rubricWeightFragment, '${rubricWeight}', 0);
+    rubricWeightFragment = replaceAll(rubricWeightFragment, 
+                                      '${Const.ParamsNames.FEEDBACK_QUESTION_RUBRIC_WEIGHT}',
+                                      'rubricWeight');
+
+    // Insert after last <th>
+    var lastWeightCell = $('#rubricWeights-' + questionNumber + ' th:last');
+    $(rubricWeightFragment).insertAfter(lastWeightCell);
```

Comment:
Just to check, although I am not sure what the value of calling <code>disallowNonNumericEntries</code> is, I think to be consistent, we need to call it here too